tests/test_minimal.py

```diff
@@ -45,9 +33,7 @@ def test_minimal_image_size(
     )
 
 
-@pytest.mark.parametrize(
-    "container", [MICRO_CONTAINER, MINIMAL_CONTAINER], indirect=True
-)
+@pytest.mark.parametrize("container", [MINIMAL_CONTAINER], indirect=True)
```

Comment:
do we need this ? can't we just use `auto_container`?

Reply:
If I use `auto_container` I get the following error message:

    E   ValueError: The test function test_fat_packages_absent is using the auto_container fixture but the parent module is not setting the 'CONTAINER_IMAGES' variable

Can somebody point me in the direction on how to solve such issues?

Reply:
Yes, you would need to define the constant:
```
CONTAINER_IMAGES = [
    MINIMAL_CONTAINER,
]
```
e.g. https://github.com/SUSE/BCI-tests/blob/main/tests/test_python.py#L7

Reply:
thnx and done